default_settings.py

```diff
@@ -33,7 +33,7 @@
     site_url = "http://127.0.0.1:4000"
 else:
     site_url = os.getenv("SITE_URL")
-    if not site_url:
+    if site_url is None:
```

Comment:
This can be reverted also. Previously I'd prefer to use `/` as base site url, but it seems conflicted with Zulip message inline href, that I drop and recover in the above commints.

Reply:
I tried setting `SITE_URL=""`, and `os.getenv` then returns an empty string instead of `None`. The previous code is safer.

Reply:
```suggestion
    if not site_url:
```

Reverted.